add simple themes function

arif-rh opened the pull request:
**_add_css_theme( $css_files )**_

This function used to easily add css files to be included in a template. With this function, we can just add css name as parameter and it will use default css path in active theme.

We can add one or more css files as parameter, either as string or array. If using parameter as string, it must use comma separator between css file name.

**Example:**

```
 1. Using string as parameter
     $this->add_css_theme( "bootstrap.min.css, style.css, admin.css" );

 2. Using array as parameter
     $this->add_css_theme( array( "bootstrap.min.css", "style.css", "admin.css" ) );
```

**_add_js_theme( $js_files )**_

This function used to easily add js files to be included in a template. With this function, we can just add js name as parameter and it will use default js path in active theme.

We can add one or more js files as parameter, either as string or array. If using parameter as string, it must use comma separator between js file name.

**Example:**

```
 1. Using string as parameter
     $this->add_js_theme( "jquery-1.11.1.min.js, bootstrap.min.js, another.js" );

 2. Using array as parameter
     $this->add_js_theme( array( "jquery-1.11.1.min.js", "bootstrap.min.js,", "another.js" ) );
```

With return as chained object we can simply do like this:

```
$this
    ->add_css_theme( "bootstrap.min.css, style.css, admin.css" )
    ->add_js_theme( "jquery-1.11.1.min.js, bootstrap.min.js, another.js" );
```

arif-rh committed:
add_css_theme and add_js_theme

add two function for easily adding css or js from current active theme

arif-rh committed:
minor change

arif-rh committed:
Merge remote-tracking branch 'origin/master' into dev-themes

JasonBaier commented:
I like it! Can you do the same for the jsi18n files? Also, can you update the readme file to reflect the new methods?

arif-rh commented:
I'd loved to do the same for the jsi18n files and update the readme. 

Now I still prepare another simple function to add css or js from out of active theme folder, after I pull this and colud be merge, I will update the readme.

arif-rh committed:
add_jsi18n_theme

arif-rh committed:
minor fix

arif-rh committed:
update README

arif-rh committed:
minor fix

arif-rh commented:
I have added function for jsi18n file and updated the README

arif-rh committed:
add set_title and set_page_header

arif-rh committed:
fix removing whitespace

$this->add_css_theme( "style.css,  bootstrap.css,  " );

this will fix error when use script like it
